code/modules/holiday/holidays.dm

```diff
@@ -314,6 +316,30 @@
 	begin_day = 14
 	begin_month = DECEMBER
 
+/datum/holiday/thanksgiving
+	name = "Thanksgiving"
+	begin_week = 4
+	begin_month = NOVEMBER
+	begin_weekday = THURSDAY
+
+/datum/holiday/columbus
+	name = "Columbus Day"
+	begin_week = 2
+	begin_month = OCTOBER
+	begin_weekday = MONDAY
+
+/datum/holiday/mother
+	name = "Mothers' Day"
+	begin_week = 2
+	begin_month = MAY
```

Comment:
Also @Shadowlight213 these days Mothering Sunday is no different from Mother's Day as the UK is quite secular these days. Mothering Sunday *IS* for all intents and purposes our Mother's Day.